### PR TITLE
Document Queues sendBatch method

### PR DESCRIPTION
**Pull request by mtlemilio**

### CODE REVIEW

File: content/queues/changelog.md [hunk 1]
```diff
@@ -11,3 +11,7 @@ weight: 11
 ### Increased per-account limits
```
Comment (a-robinson) [inline content/queues/changelog.md:11]: Should we include a second-level header for the new change like Matt did for this one?

File: content/queues/javascript-apis.md [hunk 2]
```diff
@@ -52,6 +52,11 @@ interface Queue<Body = any> {
   - Sends a message to the Queue. The body can be any type supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#supported_types), as long as its size is less than 128 KB.
   - When the promise resolves, the message is confirmed to be written to disk.
 
+- {{<code>}}sendBatch(body{{<param-type>}}any[]{{</param-type>}}){{</code>}} {{<type>}}Promise\<void>{{</type>}}
```
Comment (a-robinson) [inline content/queues/javascript-apis.md]: Same comment on the parameter type here as in the runtime types PR.

File: content/queues/javascript-apis.md [hunk 3]
```diff
@@ -52,6 +52,11 @@ interface Queue<Body = any> {
...
+
+  - Sends a batch of messages to the Queue. Each item in the array must be supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#supported_types). You can send up to 100 messages at a time, though items are limited to 128 KB each, and the total size of the array cannot exceed 256 KB.
```
Comment (a-robinson) [inline content/queues/javascript-apis.md]: Kate will probably mention this once this PR moves to be against the main docs repo, but if you want to clean it up sooner then I think we generally avoid using the second person. The last sentence here could be rephrased to something like "A batch can contain up to 100 messages, though [...]`

**Review by a-robinson — COMMENTED** [review]
I assume it was intentional that you put this PR against @elithrar 's repo because the changelog isn't on the main one yet?

File: content/queues/changelog.md (hunk 1 above)
Comment (mtlemilio) [reply content/queues/changelog.md:11]: Good call.

**Comment by mtlemilio** [comment]
Opened https://github.com/cloudflare/cloudflare-docs/pull/7023 for better visibility.

**Pull request closed without merging**